src/two/index.html

```diff
@@ -42,12 +42,12 @@
 
   <div id="olbanner">
     <a class="openlayersbanner" href="/two/" >
-      <img class="openlayersbannerimg" src="/two/images/OpenLayers.trac.png" style="height: 44px; width: 49px;border:0px" alt="OpenLayers" /> OpenLayers 2
+      <img class="openlayersbannerimg" src="/two/images/OpenLayers..png" style="height: 44px; width: 49px;border:0px" alt="OpenLayers" /> OpenLayers 2
```

Comment:
I think this url was correct: https://openlayers.org/two/images/OpenLayers.trac.png